server/routes/induction/common/inPrisonTrainingController.ts

```diff
@@ -0,0 +1,40 @@
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import type { InductionDto } from 'inductionDto'
+import type { InPrisonTrainingForm } from 'inductionForms'
+import InductionController from './inductionController'
+import InPrisonTrainingView from './inPrisonTrainingView'
+import InPrisonTrainingValue from '../../../enums/inPrisonTrainingValue'
+
+/**
+ * Abstract controller class defining functionality common to both the Create and Update Induction journeys.
+ */
+export default abstract class InPrisonTrainingController extends InductionController {
```

Comment:
Here's the actual functionality (along with `InPrisonTrainingUpdateController`). With hindsight, I probably should have split this out into a separate PR and done the renaming first, but it's a bit late now! 🤦‍♂️